pkg/api/apis.go

```diff
@@ -0,0 +1,117 @@
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package api
+
+import (
+	"github.com/dynatrace/dynatrace-configuration-as-code/internal/maps"
+	"strings"
+)
+
+type APIs map[string]*API
+
+func NewApis() APIs {
+	return getAPIs(configEndpoints)
+}
+
+func NewV1Apis() APIs {
+	return getAPIs(configEndpointsV1)
+}
+
+func getAPIs(fromApiInputs APIs) APIs {
+	apis := make(APIs)
+	for id, a := range fromApiInputs {
+		apis[id] = newAPI(id, a)
+		//a.copy().setID(id).applyRules(nameOfGetAllResponse, singleConfigurationApi)
+		//newAPI(*a, setId(id), nameOfGetAllResponse, singleConfigurationApi) //order of applying roles is important!!!
+	}
+
+	return apis
+}
```

Comment:
I think to have a "read-only" view of APIs, we can change the type from `map[string]*API` to `map[string]API`